### DISCUSSION TITLE
Add support for importing full DID documents through API endpoint in wallet-kit | ssi-kit

### DISCUSSION THREAD

**Discussion by taminobaumann** [walt-id/.github#39] [Ideas / Feature Requests]
Currently, the wallet-kit API only allows importing a DID document by providing the DID which then gets resolved. As a user, I want the ability to import a full DID document directly through the API endpoint (/issuer-api/{tenantId}/config/did/import). Similar to the CLI command `did import`.

**Comment by philpotisk** [comment]
I guess this feature would work together with the "key import" command, so you would first import a key pair and then the associated DID, right?

**Reply by taminobaumann** [reply]
Yes, this is another way to achieve it.
